### PR TITLE
Ui

### PR DESCRIPTION
**Pull request by mariakatusiime**

### CODE REVIEW

File: UI/admin/dishes.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (mozzy22) [inline UI/admin/dishes.html:1]: add a `lang` attribute in the opening `html` tag.   seems like your using html 4 . u need to use html 5 , probably your missing out on some newfeatures

**Comment by mozzy22** [comment]
can u please add your ghpages link and we have a look at it

**Comment by mariakatusiime** [comment]
How do I add the gh-pages link.

**Comment by Phionamary** [comment]
@mariakatusiime please add descriptions for your PR.  Makes it easier for us to understand what we are reviewing. 
Also about gh-pages,  try this: https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/

File: UI/admin/dishes.html (hunk 1 above)
Comment (mariakatusiime) [reply UI/admin/dishes.html:1]: Thanks for the feeback, let me work on that

**Comment by ongebo** [comment]
I think since these are still templates, you don't need to do form validation, when a user clicks Sign Up or Sign In, they should be redirected to a subsequent page regardless of whether or not they entered any data into the forms

**Pull request merged**